integration-tests/tests/billing-projects.js

```diff
@@ -89,14 +101,18 @@ const setAjaxMockValues = async (testPage, ownedBillingProjectName, notOwnedBill
     })
   }, _.range(0, numExtraWorkspaces))
 
-  const projectListResult = [{
+  const projectListResult = _.compact([{
```

Comment:
@MatthewBemis I pushed up a couple of test improvements. The compact is needed here else you end up with `false` as the second of 3 entries, which causes a React rendering warning.